### PR TITLE
[logtalk/en] Initial version of the Logtalk language tutorial

### PR DESCRIPTION
**Pull request by pmoura**
- [x] PR touches only one file (or a set of logically related files with similar changes made)
- [x] Content changes are aimed at *intermediate to experienced programmers* (this is a poor format for explaining fundamental programming concepts)
- [x] YAML Frontmatter formatted according to [CONTRIBUTING.md](https://github.com/adambard/learnxinyminutes-docs/blob/master/CONTRIBUTING.markdown)
- [x] Seriously, look at it now. Watch for quotes and double-check field names.

### CODE REVIEW

File: logtalk.html.markdown [hunk 1]
```diff
@@ -0,0 +1,513 @@
...
+	instantiates(class)).
+
+	:- public([
+		age/1, name/1
+	]).
+
+	% a default value for age/1
+	age(42).
+
+:- end_object.
```
Comment (pmoura) [inline logtalk.html.markdown]: You get an error. `person` plays the role of class and, as such, declares the predicates `name/1` and `age/1` for its instances. It doesn't define those predicates for itself. Note also that the meta-class of `person` only declares (and defines) a `new/2` predicate, which thus is the only valid message to `person`.

You can trust that all code examples in the tutorial are correct (I'm the author of the Logtalk programming language).

Comment (vendethiel) [inline logtalk.html.markdown]: I know you are. I'm just asking you to put up an example demonstrating it's an error (assuming I didn't miss an existing one)

Comment (pmoura) [inline logtalk.html.markdown]: That's a good idea. I added an example in an earlier section and improve the comment describing the class. Thanks for the feedback.

**Comment by vendethiel** [comment]
The rest seems fine to me. I trust you with the technical details. Tell me when you want me to merge.

**Comment by pmoura** [comment]
Thanks for approving the pull request. Looking forward to see the tutorial published in the website. I would appreciate if you could merge as soon as feasible.

**Comment by vendethiel** [comment]
That's around right now. Thanks! :cake:

**Comment by pmoura** [comment]
I see it listed in the home page but I get a `404 Not Found` error when clicking on the link.

**Pull request merged**